govc/session/login.go

```diff
@@ -65,8 +64,7 @@ func (cmd *login) Register(ctx context.Context, f *flag.FlagSet) {
 	f.BoolVar(&cmd.long, "l", false, "Output session cookie")
 	f.StringVar(&cmd.ticket, "ticket", "", "Use clone ticket for login")
 	f.StringVar(&cmd.cookie, "cookie", "", "Set HTTP cookie for an existing session")
-	f.StringVar(&cmd.token, "token", "", "Use SAML token for login")
-	f.StringVar(&cmd.actas, "actas", "", "Issue SAML token with this token's identity")
+	f.StringVar(&cmd.token, "token", "", "Use SAML token for login or issue identity")
```

Comment:
It took me a few re-reads to understand what was meant by "or issue identity". Perhaps "Use SAML token for login or as issue identity" would be slightly clearer to some users?